docs/gke.md

```diff
@@ -0,0 +1,192 @@
+        customresourcedefinition.apiextensions.k8s.io/perconaservermongodbbackups.psmdb.percona.com serverside-applied
+        customresourcedefinition.apiextensions.k8s.io/perconaservermongodbrestores.psmdb.percona.com serverside-applied
+        role.rbac.authorization.k8s.io/percona-server-mongodb-operator serverside-applied
+        serviceaccount/percona-server-mongodb-operator serverside-applied
+        rolebinding.rbac.authorization.k8s.io/service-account-percona-server-mongodb-operator serverside-applied
+        deployment.apps/percona-server-mongodb-operator serverside-applied
+        ```
+
+2. The operator has been started, and you can deploy your MongoDB cluster:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/cr.yaml
+    ```
+
+    ??? example "Expected output"
+
+        ``` {.text .no-copy}
+        perconaservermongodb.psmdb.percona.com/my-cluster-name created
+        ```
+
+    !!! note
+
+        This deploys default MongoDB cluster configuration, three mongod, three mongos, and
+        three config server instances. Please see [deploy/cr.yaml](https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/cr.yaml)
+        and [Custom Resource Options](operator.md#operator-custom-resource-options)
+        for the configuration options. You can clone the repository with all
+        manifests and source code by executing the following command:
+
+        ```bash
+        $ git clone -b v{{ release }} https://github.com/percona/percona-server-mongodb-operator
+        ```
+
+        After editing the needed options, apply your modified `deploy/cr.yaml` file as follows:
+
+        ```bash
+        $ kubectl apply -f deploy/cr.yaml
+        ```
+
+    The creation process may take some time. When the process is over your
+    cluster will obtain the `ready` status. You can check it with the following
+    command:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl get psmdb
+    ```
+
+    ??? example "Expected output"
+
+        ``` {.text .no-copy}
+        NAME              ENDPOINT                                           STATUS   AGE
+        my-cluster-name   my-cluster-name-mongos.default.svc.cluster.local   ready    5m26s
+        ```
+
+??? note "You can also track the creation process in Google Cloud console via the Object Browser"
+
+    When the creation process is finished, it will look as follows:
+
+    ![image](assets/images/gke-quickstart-object-browser.svg)
+
+
+## Verifying the cluster operation
+
+It may take ten minutes to get the cluster started. When `kubectl get psmdb`
+command finally shows you the cluster status as `ready`, you can try to connect
+to the cluster.
+
+{% include 'assets/fragments/connectivity.txt' %}
+
+
+## Removing the GKE cluster
+
+There are several ways that you can delete the cluster.
```

Comment:
We state there are several ways but show only one. Either delete this sentence or provide other ways of deleting it

Reply:
Fixed.

Reply:
Fixed.